configuration/scripts/machines/Macros.freya_intel

```diff
@@ -0,0 +1,75 @@
+#==============================================================================
+# Makefile macros for DMI Freya based on ECCC banting
+#==============================================================================
+# For use with intel compiler
+#==============================================================================
+
+#INCLDIR := -I. -I/usr/include
+#SLIBS   :=
+
+#--- Compiler/preprocessor ---
+FC         := ftn 
+CC         := cc
+CXX        := CC
+CPP        := /usr/bin/cpp
+CPPFLAGS   := -P -traditional  # ALLOW fortran double backslash "\\"
+SCC   := gcc
+SFC   := ftn
+
+CPPDEFS    := -DFORTRANUNDERSCORE ${ICE_CPPDEFS}
+CFLAGS     := -c -O2 -fp-model precise
+
+# Additional flags
+FIXEDFLAGS := -132
+FREEFLAGS  := -FR
+FFLAGS     := -fp-model source -convert big_endian -assume byterecl -ftz -traceback -no-wrap-margin
+#-xHost
+
+ifeq ($(ICE_BLDDEBUG), true)
+  FFLAGS     += -O0 -g -check -fpe0 -ftrapuv -fp-model except -check noarg_temp_created
+# -heap-arrays 1024 
+else
+  FFLAGS     += -O2
+endif
+LD         := $(FC)
+LDFLAGS    := $(FFLAGS) -v
+#ifeq ($(ICE_BLDDEBUG), true)
+#FFLAGS     := -O0 -g -check uninit -check bounds -check pointers -fpe0 -check noarg_temp_created
+#FFLAGS     := -g -O0 -traceback -fp-model precise -fp-stack-check -fpe0
+#else
+#FFLAGS     := -r8 -i4 -O2 -align all -w -ftz -assume byterecl
+# FFLAGS     := -O2 -fp-model precise -assume byterecl -ftz -traceback -xHost
+#endif
+# Preprocessor flags
+#CPPDEFS    := -DLINUX $(ICE_CPPDEFS)
+
+# Linker flags
+
+# Additional flags
+
+ifeq ($(DITTO), yes)
```

Comment:
the is probably obsolete.. greping either 'DITTO' and 'REPRODUCIBLE' does not return any result for me..